<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(64, 64, 64);</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 30pt &quot;나눔명조 ExtraBold&quot;;

color: rgb(146, 208, 80);
background-color: rgb(64, 64, 64);</string>
   </property>
   <property name="text">
    <string>Hello, Welcome to Main</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_logout">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;나눔명조 ExtraBold&quot;;

border: 1.7px solid rgb(45, 45, 185);
background-color: rgb(191, 191, 191);</string>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_push_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;나눔명조 ExtraBold&quot;;

border: 1.7px solid rgb(45, 45, 185);
background-color: rgb(191, 191, 191);</string>
   </property>
   <property name="text">
    <string>File Encrypt</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_push_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;나눔명조 ExtraBold&quot;;

border: 1.7px solid rgb(45, 45, 185);
background-color: rgb(191, 191, 191);</string>
   </property>
   <property name="text">
    <string>File Decrypt</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>551</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>title</zorder>
  <zorder>btn_logout</zorder>
  <zorder>btn_push_1</zorder>
  <zorder>btn_push_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
